### PR TITLE
fix: Typings for OTR message sending

### PR DESCRIPTION
**Pull request by bennyn**

### CODE REVIEW

File: src/main/conversation/ConversationAPI.ts [hunk 1]
```diff
@@ -227,9 +227,20 @@ export default class ConversationAPI {
...
    */
-  public postOTRMessage(clientId: string, conversationId: string, messageData: NewOTRMessage): Promise<ClientMismatch> {
+  public postOTRMessage(
+    clientId: string,
+    conversationId: string,
+    messageData?: NewOTRMessage,
+  ): Promise<ClientMismatch> {
     const hasContent = !!messageData.data;
 
+    if (!messageData) {
```
Comment (gregor) [inline src/main/conversation/ConversationAPI.ts]: Shouldn't this go above the `hasContent` const as it fails without a default value.

File: src/main/connection/ConnectionAPI.ts [hunk 2]
```diff
@@ -2,7 +2,7 @@ import {AxiosRequestConfig, AxiosResponse} from 'axios';
 
 import {HttpClient} from '../http';
 import {ClientPreKey, PreKeyBundle} from '../auth';
-import {PublicClient} from '../client';
+import {PublicClient} from '../client/index';
```
Comment (ffflorian) [inline src/main/connection/ConnectionAPI.ts]: We don't need the `index`, as it is assumed by TS.

File: src/main/conversation/NewOTRMessage.ts [hunk 3]
```diff
@@ -1,7 +1,7 @@
 import {OTRRecipients} from '../conversation';
 
 interface NewOTRMessage {
-  data?: any; // Use 'string' for JSON, default is Protobuf
+  data?: any; // Defaults to Protocol Buffers. Use type 'string' for JSON.
```
Comment (ffflorian) [inline src/main/conversation/NewOTRMessage.ts]: Didn't we want to call it "Protocol Buffer" (singular)?

File: package.json [hunk 4]
```diff
@@ -68,7 +68,7 @@
     "version": "yarn build:node && yarn pack:demo && yarn dist:demo",
     "postversion": "git push && git push --tags && yarn pack:browser",
     "precommit": "lint-staged",
-    "prettier": "prettier --single-quote --trailing-comma=\"all\" --no-bracket-spacing --print-width=120 --write \"src/**/*.ts\"",
+    "prettier": "prettier --single-quote --trailing-comma=\"all\" --no-bracket-spacing --print-width=1337 --write \"src/**/*.ts\"",
```
Comment (ffflorian) [inline package.json]: :scream:

**Pull request merged**